packages/beeq/src/components/progress/_storybook/bq-progress.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+import mdx from './bq-progress.mdx';
+import { PROGRESS_BORDER_SHAPE, PROGRESS_THICKNESS, PROGRESS_TYPE } from '../bq-progress.types';
+
+const meta: Meta = {
+  title: 'Components/Progress',
+  component: 'bq-progress',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    value: { control: 'number' },
+    indeterminate: { control: 'boolean' },
+    thickness: { control: 'select', options: [...PROGRESS_THICKNESS] },
+    type: { control: 'select', options: [...PROGRESS_TYPE] },
+    'border-shape': { control: 'select', options: [...PROGRESS_BORDER_SHAPE] },
+    label: { control: 'boolean' },
+    'enable-tooltip': { control: 'boolean' },
+  },
+  args: {
+    value: 0,
+    indeterminate: false,
+    thickness: 'medium',
+    type: 'default',
+    'border-shape': 'rounded',
+    label: false,
+    'enable-tooltip': false,
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => html`
+  <div class="w-64">
+    <bq-progress
+      value=${args.value}
+      indeterminate=${args.indeterminate}
+      thickness=${args.thickness}
+      type=${args.type}
+      border-shape=${args['border-shape']}
+      ?label=${args.label}
+      ?enable-tooltip=${args['enable-tooltip']}
+    ></bq-progress>
+  </div>
+`;
+
+export const Default: Story = {
+  render: Template,
+  args: {
+    value: 10,
+  },
+};
+
+export const MediumThickness: Story = {
```

Comment:
The name of this story is misleading, we're showing here the large-thickness example

```suggestion
export const LargeThickness: Story = {
```